spec/services/proofing/lexis_nexis/phone_finder/proofing_spec.rb

```diff
@@ -28,9 +28,19 @@
 
   describe '#proof' do
     context 'when the response is a success' do
-      it 'is a successful result' do
+      it 'is a successful rdp1 result' do
         stub_request(:post, verification_request.url).
-          to_return(body: LexisNexisFixtures.phone_finder_success_response_json, status: 200)
+          to_return(body: LexisNexisFixtures.phone_finder_rdp1_success_response_json, status: 200)
+
+        result = instance.proof(applicant)
+
+        expect(result.success?).to eq(true)
+        expect(result.errors).to eq({})
+      end
+
+      it 'is a successful rdp2 result' do
+        stub_request(:post, verification_request.url).
+          to_return(body: LexisNexisFixtures.phone_finder_rdp2_success_response_json, status: 200)
 
         result = instance.proof(applicant)
```

Comment:
This all looks good. Should we throw an expectation below somewhere to confirm that JSONified result doesn't include one of the PII fields?